spring-cloud-gcp-logging/src/main/java/org/springframework/cloud/gcp/logging/StackdriverServiceContext.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2017-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.logging;
+
+/**
+ * This class provides the service context data for stackdriver error reporting.
+ *
+ * Reference:  https://cloud.google.com/error-reporting/reference/rest/v1beta1/ServiceContext
+ *
+ * @author Stefan Dieringer
+ */
+public class StackdriverServiceContext {
+
+	private String service;
+
+	private String version;
```

Comment:
What about `resourceType`?

Reply:
As i understand from the documentation, the ressourceType is set automatically and must not be set by error reporting. https://cloud.google.com/error-reporting/reference/rest/v1beta1/ServiceContext.

Reply:
Ok, fair enough.